tasmota/xdrv_23_zigbee_5_converters.ino

```diff
@@ -1601,6 +1601,9 @@ void ZCLFrame::postProcessAttributes(uint16_t shortaddr, JsonObject& json) {
       }
 
       Z_Device & device = zigbee_devices.getShortAddr(shortaddr);
+
+      device.last_seen = Rtc.utc_time;
```

Comment:
I believe this should be higher in the message flow, for example in `Z_IncomingMessage()` which means adding a method `lastSeenNow()` to `Z_Device` class, similarly to `setLQI()` and call it whenever `setLQI()` is called.